configuration-resolver/trino-configuration-resolvers/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we use the `plugin` directory or have a new directory for the `configuration-resolvers` ?